src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+          rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <span class="logo__container">
+          <a class="logo__link"
+             href="index.html">
```

Comment:
Prefer linking to `/` for homepage rather `index.html`. On most pages `index.html` is not a part of url
```suggestion
             href="/">
```